VK/VK/Model/Group.swift

```diff
@@ -0,0 +1,8 @@
+// Group.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+struct Group {
```

Comment:
```suggestion

struct Group {
```
Добавь комент который требует линтер!